samples/drawing/drawing.cpp

```diff
@@ -822,7 +822,7 @@ void MyCanvas::DrawText(wxDC& dc)
     dc.DrawRotatedText( wxT("That is text"), 20, 10, -45 );
 
     // use wxSWISS_FONT and not wxNORMAL_FONT as the latter can't be rotated
-    // under Win9x (it is not TrueType)
+    // under MSW (it is not TrueType)
```

Comment:
Yes, that is what I think too.
On Win7 `wxNORMAL_FONT` does rotate fine. It would still need the call to `SetFont(*wxNORMAL_FONT)` though as the DC will not pick up the font from its parent window. Otherwise I don't know which font would better be given as example.
...nor how thing behave on XP.